_install-and-configure/install-opensearch/windows.md

```diff
@@ -66,15 +59,15 @@ An OpenSearch node in its default configuration (with demo certificates and user
          {% include copy.html %}
 
       1. Run the batch script.
-         ```bat
-         .\opensearch-windows-install.bat
-         ```
          {% include copy.html %}
          For OpenSearch 2.12 or greater, set a new custom admin password before installation using the following command:
          ```bat
          > set OPENSEARCH_INITIAL_ADMIN_PASSWORD=<custom-admin-password>
          ```
          {% include copy.html %}
```

Comment:
setting the env variable should come before running the bat file, but I am not seeing the copy on the website, so not sure what's going on here. Can somebody from the doc team help out here?

Reply:
Also can somebody help me understand how this plays into what version is shown when you select different versions from the dropdown? If it would only show up for 2.12 onwards anyway, maybe we can get rid of the copy and just leave it as a requirement from now on?